dss/collections/__init__.py

```diff
@@ -0,0 +1,32 @@
+import os
+
+from dss import dynamodb  # type: ignore
+
+
+collection_db_table = f"dss-collections-db-{os.environ['DSS_DEPLOYMENT_STAGE']}"
+
+
+def put_collection(owner: str, uuid: str, permission_level: str = 'owner'):
+    dynamodb.put_item(table=collection_db_table,
+                      hash_key=owner,
+                      sort_key=uuid,
+                      value=permission_level,
+                      dont_overwrite='sort_key')
+
+
+def get_collection_uuids_for_owner(owner: str) -> list:
+    items = dynamodb.get_primary_key_items(table=collection_db_table,
+                                           key=owner,
+                                           return_key='sort_key')
+    return [item for item in items]
+
+
+def get_all_collection_uuids():
```

Comment:
Redundant with `dss.collections. get_all_collection_uuids`?

Reply:
`get_all_collection_uuids` gets all uuids (all owners) in the table for repopulating it from the `update_collections_db.py` script.  `get_collection_uuids_for_owner` fetches only the current owner related uuids for the API.

Does that address your comment?